tests/core/atoms/test_skzcam.py

```diff
@@ -1,146 +1,555 @@
 from __future__ import annotations
 
-from copy import deepcopy
 from pathlib import Path
 
 import numpy as np
 import pytest
-from ase.build import bulk, fcc100, molecule
-from ase.io import read
 from ase import Atoms
+from ase.io import read
+
```

Comment:
Note that you can add a pytest marker to skip a test if a given import is not found (like chemsh): https://docs.pytest.org/en/stable/how-to/skipping.html#skipping-on-a-missing-import-dependency